profile/2020-09-01-hybrid.json

```diff
@@ -0,0 +1,612 @@
+        }
+      ]
+    },
+    "microsoft.eventhubs": {
+      "2018-01-01-preview": [
+        {
+          "resourceType": "availableClusterRegions",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/AvailableClusterRegions-preview.json"
+        },
+        {
+          "resourceType": "clusters",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json"
+        },
+        {
+          "resourceType": "namespaces",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/namespaces-preview.json"
+        }
+      ],
+      "2017-04-01": [
+        {
+          "resourceType": "checkNameAvailability",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json"
+        },
+        {
+          "resourceType": "namespaces/authorizationRules",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json"
+        },
+        {
+          "resourceType": "namespaces/eventhubs",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json"
+        },
+        {
+          "resourceType": "namespaces/eventhubs/authorizationRules",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json"
+        },
+        {
+          "resourceType": "namespaces/eventhubs/consumerGroups",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json"
+        },
+        {
+          "resourceType": "operations",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json"
+        },
+        {
+          "resourceType": "sku",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json"
+        }
+      ],
+      "2015-08-01": [
+        {
+          "resourceType": "checkNamespaceAvailability",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2015-08-01/EventHub.json"
```

Comment:
There is no "checkNamespaceAvailability" related content in this file. It only exists in version 2014-09-01.